### PR TITLE
Fix for WFMP-257, Glow support ignore channels

### PR DESCRIPTION
**Pull request by jfdenise**
Issue: https://issues.redhat.com/browse/WFMP-257

### CODE REVIEW

File: plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java [hunk 1]
```diff
@@ -89,6 +97,10 @@ public ChannelMavenArtifactRepositoryManager(List<ChannelConfiguration> channels
         this.system = system;
     }
 
+    public ChannelSession getChannelSession() {
```
Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java:100]: Does this need to be public?

File: tests/standalone-tests/pom.xml [hunk 2]
```diff
@@ -82,7 +82,10 @@
             <artifactId>parsson</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.wildfly.channel</groupId>
+            <artifactId>channel-core</artifactId>
```
Comment (jamezp) [inline tests/standalone-tests/pom.xml:87]: This looks like it should have `<scope>test</scope>`.

File: plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java (hunk 1 above)
Comment (jfdenise) [reply plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java]: This was a left over, I removed the method.

File: tests/standalone-tests/pom.xml (hunk 2 above)
Comment (jfdenise) [reply tests/standalone-tests/pom.xml:87]: Good, fixed.

**Comment by jamezp** [comment]
I'm not sure I follow what this means by "Ignore channels". Could you explain that a bit please @jfdenise?

**Comment by jfdenise** [comment]
@jamezp , sure. In the Glow upgrade we got: https://github.com/wildfly/wildfly-glow/commit/3c2b34ed5ab5b152012bdac35db6238b06fdc053
Glow will use the configured channel to resolve the feature-packs. Without this upgrade, the hard coded versions of feature-packs known by Glow are used.
The changes in the ChannelMavenArtifactManager, are fixes found adding tests to cover channels with upstream feature-packs.

**Comment by jfdenise** [comment]
@jamezp , yes, glow 1.1.1 contains fixes that allow for correct channel handling.

**Review by jamezp — APPROVED** [review]

**Pull request merged**